### PR TITLE
Add release notes for 1.30.0

### PR DESCRIPTION
**Pull request by ikhoon**

### CODE REVIEW

**Comment by ikhoon** [comment]
image

File: site/src/pages/release-notes/1.30.0.mdx [hunk 1]
```diff
@@ -0,0 +1,164 @@
+---
+date: 2024-08-08
```
Comment (minwoox) [inline site/src/pages/release-notes/1.30.0.mdx:2]: Don't forget to change the date. 😉 
```suggestion
date: 2024-08-12
```

File: site/src/pages/release-notes/1.30.0.mdx [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+  a write operation times out. #4935 #5156
+
+## 📃 Documentation
+
+- Added a gRPC reverse proxy server example. #2353 #5722
+
+## ☢️ Breaking changes
+
+- Modified the behavior of <type://Endpoint#withAttrs(Attributes)> to merge the original <type://Attributes> 
+  with the new one.
```
Comment (minwoox) [inline site/src/pages/release-notes/1.30.0.mdx:120]:
```suggestion
  with the new one. #5802
```

**Review by minwoox — APPROVED** [review]
👍 👍 👍

**Review by jrhee17 — APPROVED** [review]
👍 👍 👍

**Pull request merged**